fiware-region-sanity-tests/tests/fiware_region_base_tests.py

```diff
@@ -28,6 +28,9 @@
 from commons.constants import *
 from novaclient.exceptions import Forbidden, OverLimit
 from datetime import datetime
+from commons.ssh_client import SSHClient
+from paramiko.ssh_exception import AuthenticationException
```

Comment:
Remove this import, so that module doesn't depend on Paramiko. Given that "commons.ssh_cllent" already imports such exception, this line could be removed and the AuthenticationException will be still available as if it was part of the SSHClient.